pynest/examples/hpc_benchmark.py

```diff
@@ -0,0 +1,426 @@
+# -*- coding: utf-8 -*-
+#
+#  hpc_benchmark.py
+#
+#  This file is part of NEST.
+#
+#  Copyright (C) 2004 The NEST Initiative
+#
+#  NEST is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  NEST is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+
+'''
+   This script produces a balanced random network of scale*11250 neurons in
+   which the excitatory-excitatory neurons exhibit STDP with
+   multiplicative depression and power-law potentiation. A mutual
+   equilibrium is obtained between the activity dynamics (low rate in
+   asynchronous irregular regime) and the synaptic weight distribution
+   (unimodal). The number of incoming connections per neuron is fixed
+   and independent of network size (indegree=11250).
+
+   This is the standard network investigated in:
+   Morrison et al (2007). Spike-timing-dependent plasticity in balanced random
+     networks. Neural Comput 19(6):1437-67
+   Helias et al (2012). Supercomputers ready for use as discovery machines for
+     neuroscience. Front. Neuroinform. 6:26
+   Kunkel et al (2014). Spiking network simulation code for petascale
+     computers. Front. Neuroinform. 8:78
+
+   A note on scaling
+   -----------------
+
+   This benchmark was originally developed for very large-scale simulations on
+   supercomputers with more than 1 million neurons in the network and
+   11.250 incoming synapses per neuron. For such large networks, input to a
+   single neuron will be little correlated and network activity will remain
```

Comment:
Correlated across neurons, right?
Perhaps change to "synaptic inputs will have low correlations across neurons"

Reply:
Yes, I think this means across neurons. This is the text from `hpc_benchmark.sli`. Should I also change it there?

Reply:
I would say, yes, though it is a minor point.